website/src/content/deployment-examples/google-cloud-storage.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Deploying to Google cloud
+---
+
+This deployment example refers to [Google cloud Static Web Apps](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#0)
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Google](https://accounts.google.com/SignUp) account.
+- A custom domain
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) you got when you configured the Custom Application in the Merchant Center.
+
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
+
+# Set up Google Cloud
+
+To host the production bundles of your Custom Application you need to create a [Cloud Storage bucket](https://console.cloud.google.com/storage) in Google Cloud and [configure](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#3) the bucket.
+
+# Configuration
+
+Depending on the access options you choose, if you want to configure permissions, select the created bucket, go to the permissions tab and click the `Add` link. For the `New principals` field, select `AllUsers` and for the `Role`, search for `Storage Object Viewer`. Click `Save` to make bucket publicly accessible.
+
+# Have a valid domain
+
+Make sure you are the owner and administrator of your domain.
+
+Merchant Center only allows accessing your Custom Application using a secure protocol (HTTPS) so your domain must support it with a valid [SSL certificate](https://support.google.com/domains/answer/7630973?hl=en&visit_id=637951191082633113-3999318342&rd=1). Google does not provide one automatically
```

Comment:
```suggestion
# Domain verification

Creating buckets with domain names requires to verify the ownership of the domain.

<Info>

Google does not provide a domain by default when using Google Cloud Storage. If you don't have an existing domain, then there are many services through which you can register a new domain, such as [Google Domains](https://domains.google.com/).

</Info>

We recommend following the instructions for [Domain-named bucket verification](https://cloud.google.com/storage/docs/domain-name-verification) to configure things properly.
```

Reply:
Thank you. 
Feedback updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2740/commits/7fd895792887039dc496fa96b8804ef3f03a97b1
I took the opportunity to emphasize some configurations on the load balancer to help link the SSL certificate to the domain.
Finally have the Custom app up and working as expected 😅

Reply:
Nice!